doc/shared/RecentChanges.rst

```diff
@@ -20,6 +20,9 @@ which should ease building SUNDIALS with LAPACK libraries that require setting
 specific linker flags e.g., MKL.
 
 **Bug Fixes**
+Fixed a `bug <https://github.com/LLNL/sundials/issues/581>`_ in 
```

Comment:
Looks like sphinx is not happy about the `bug` target name for the link (I did not realize they had to be unique) and `SUNMatScaleAddI_Sparse` is not recognized as a function (I think the docs for it are in the old format where we do not fully write out the docs for the functions which implement the generic operations).

Reply:
Looks like I needed to use an anonymous link which doesn't make a target. It uses a double underscore instead of a single. I went ahead and switched to that for all external links in the changelog.

https://stackoverflow.com/a/27435441